include/bluetooth/mesh/sensor_types.h

```diff
@@ -44,6 +44,336 @@ extern "C" {
+ *  - Range: 0 to 100.0
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_percentage_8;
+
+/** Percentage 16
+ *  - Unit: Percent
+ *  - Encoding: 16 bit unsigned scalar (Resolution: 0.01)
+ *  - Range: 0 to 100.0
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_percentage_16;
+
+/** Percentage delta trigger
+ *  - Unit: Percent
+ *  - Encoding: 16 bit unsigned scalar (Resolution: 0.01)
+ *  - Range: 0 to 655.35
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_percentage_delta_trigger;
+/** @} */
+
+/** @defgroup bt_mesh_sensor_formats_environmental Environmental sensor formats
+ *  @{
+ */
+
+/** Temp 8
+ *  - Unit: Celsius
+ *  - Encoding: 8 bit signed scalar (Resolution: 0.5)
+ *  - Range: -64.0 to 63.0
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_temp_8;
+
+/** Temp
+ *  - Unit: Celsius
+ *  - Encoding: 16 bit signed scalar (Resolution: 0.01)
+ *  - Range: -327.67 to 327.67
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_temp;
+
+/** Co2 concentration
+ *  - Unit: Ppm
+ *  - Encoding: 16 bit unsigned scalar (Resolution: 1.0)
+ *  - Range: 0 to 65534
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_co2_concentration;
+
+/** Noise
+ *  - Unit: Db
+ *  - Encoding: 8 bit unsigned scalar (Resolution: 1.0)
+ *  - Range: 0 to 254
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_noise;
+
+/** Voc concentration
+ *  - Unit: Ppb
+ *  - Encoding: 16 bit unsigned scalar (Resolution: 1.0)
+ *  - Range: 0 to 65534
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_voc_concentration;
+
+/** Wind speed
+ *  - Unit: Mps
+ *  - Encoding: 16 bit unsigned scalar (Resolution: 0.01)
+ *  - Range: 0 to 655.35
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_wind_speed;
+
+/** Temp 8 wide
+ *  - Unit: Celsius
+ *  - Encoding: 8 bit signed scalar (Resolution: 1.0)
+ *  - Range: -128 to 127
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_temp_8_wide;
+
+/** Gust factor
+ *  - Unit: Unitless
+ *  - Encoding: 8 bit unsigned scalar (Resolution: 0.1)
+ *  - Range: 0 to 25.5
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_gust_factor;
+
+/** Magnetic flux density
+ *  - Unit: Microtesla
+ *  - Encoding: 16 bit signed scalar (Resolution: 0.1)
+ *  - Range: -3276.8 to 3276.7
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_magnetic_flux_density;
+
+/** Pollen concentration
+ *  - Unit: Concentration
+ *  - Encoding: 24 bit unsigned scalar (Resolution: 1.0)
+ *  - Range: 0 to 16777215
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_pollen_concentration;
+
+/** Pressure
+ *  - Unit: Pascal
+ *  - Encoding: 32 bit unsigned scalar (Resolution: 0.1)
+ *  - Range: 0 to 429496729.5
+ */
+extern const struct bt_mesh_sensor_format bt_mesh_sensor_format_pressure;
+
+/** Rainfall
+ *  - Unit: Metre
```

Comment:
Meters
(US English)

Reply:
This is based on the `unit` in `sensor_types.c`, where this is line 80:

```
UNIT(metre) = { "Metre", "m" };
```

Should I change it there as well? There are several units there, all spelled in UK spelling.

Reply:
If changed, they should be changed in both places. But maybe we could do that as a cleanup task of the values in sensor_types.c.

Reply:
I'm ok with this being fixed in a separate cleanup task.